soc/xtensa/intel_adsp/ace/multiprocessing.c

```diff
@@ -20,6 +20,11 @@
 #include <zephyr/irq.h>
 #include <zephyr/cache.h>
 
+#if CONFIG_ADSP_DYNAMIC_CLOCK_SWITCHING
+#include <zephyr/pm/policy.h>
+#include <pm_data.h>
+#endif
```

Comment:
Drop the conditional for more coverage, see below. This gives Zephyr CI and others a chance to find issues before other CIs do.

BTW don't forget to submit this to SOF CI with a one-line `sof/west.yml` change to catch problems earlier, unlike what happened in

- https://github.com/zephyrproject-rtos/zephyr/pull/67393#issuecomment-1889640994

Reply:
The pull request to SOF was already made earlier. The link is in the description.